haproxy_exporter.go

```diff
@@ -211,6 +213,10 @@ func NewExporter(uri string, selectedServerMetrics map[int]*prometheus.GaugeVec,
 			42: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "4xx"}),
 			43: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "5xx"}),
 			44: newBackendMetric("http_responses_total", "Total of HTTP responses.", prometheus.Labels{"code": "other"}),
+			58: newBackendMetric("http_queue_time", "Avg. HTTP queue time for last 1024 successful connections.", nil),
```

Comment:
This should include a unit, and if I understand the metric correctly, should have a more apt name. Maybe something like:

`http_average_queue_time_seconds`

I don't know if it's worth including in the metric name the fact that it is an exponential moving average (do we know this for sure?), and over the last 1024 connections.

Same for the other new metrics here.